rules/ruleset.go

```diff
@@ -502,10 +511,37 @@ func NewRuleSet(version int, rs *schema.RuleSet, opts Options) (RuleSet, error)
 	}, nil
 }
 
+// NewRuleSetFromSchema creates a new RuleSet from a schema object.
+func NewRuleSetFromSchema(version int, rs *schema.RuleSet, opts Options) (RuleSet, error) {
+	return newRuleSetFromSchema(version, rs, opts)
+}
+
+// NewMutableRuleSetFromSchema creates a new MutableRuleSet from a schema object
+func NewMutableRuleSetFromSchema(version int, rs *schema.RuleSet) (MutableRuleSet, error) {
+	// Takes a blank Options stuct because none of the mutation functions need the options.
+	return newRuleSetFromSchema(version, rs, NewOptions())
+}
+
+// InitRuleSet returns an empty ruleset to be used with a new namespace
+func InitRuleSet(namespaceName string, opts Options, meta UpdateMetadata) MutableRuleSet {
```

Comment:
Yes, its not being used in this PR but this function will be necessary for the caller to make a brand new ruleset when a namespace is created. Will rename and remove opts.